qemu/tests/hpt_miscellaneous.py

```diff
@@ -88,6 +89,7 @@ def run(test, params, env):
         # For HPT reszing after hotplug memory
         hpt_mem = MemoryHotplugTest(test, params, env)
         hpt_mem.hotplug_memory(vm, "hpt_mem")
+        time.sleep(int(params.get("wait_time", 20)))
```

Comment:
I thought it is better to check if the size increase is equal to your except in a certain duration, don't need to sleep the $wait_time sec in every time.

Reply:
Per you,I changed the code accordingly,could you give a look again ? Thanks a lot